### DISCUSSION TITLE
AppleStore - Deliver end with success but no app is uploaded

### DISCUSSION THREAD

**Discussion by defaultbr** [fastlane/fastlane#21008] [Q&A] [answered]
Hi, I have this lane:

```
  lane :ios_store do |options|
    deliver(
      api_key: get_api_key,
      app: "64xxxxx937",
      submit_for_review: false,
      automatic_release: false,
      force: true, 
      skip_metadata: true,
      skip_screenshots: true,
      skip_binary_upload: false,
      run_precheck_before_submit: false
    ) 
  end  
```

The pipeline run fast as hell and say that the file was uploaded, but it never show up on the apple developer portal:

![image](https://user-images.githubusercontent.com/5432351/214132295-eeef6ef0-1057-415f-bc8e-facf57dce9bd.png)

Pipeline ends in 13 seconds, lol, of course its not uploading anything

The testflight version upload takes almost 2 minutes, only the upload to appstore with deliver that takes 13 seconds.

Maybe I'm missing something?

ps: the app is in draft (never sent to review on the developer portal), can be this? Is there any documentation about this rule?

**Comment by ghost** [comment, marked as answer]
try to run the command: 
fastlane pilot builds 

Will list the uploaded builds and the Pending ones.
See if the expected is on the pending.

For us it happens that after 6 hours still on pending 
Some of the apps are on pending others succeeded  after more hours

**Comment by ghost** [comment]
now looks the upload is working as expected
